src/Entity/User.php

```diff
@@ -35,6 +42,59 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
      */
     private string $password;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank(
+     *     message = "Le prénom est obligatoire")
+     */
+    private string $firstname;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank(
+     *     message = "Le nom de famille est obligatoire")
+     */
+    private string $lastname;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * @Assert\NotBlank(
+     *     message = "Le genre est obligatoire")
+     */
```

Comment:
Your form seems to offer 4 choices (including null)
Your Constraint may not be consistant with that.
You may take a look at Choices constraints